internal/driver/glfw/window_desktop.go

```diff
@@ -141,6 +142,23 @@ func (w *window) CenterOnScreen() {
 	}
 }
 
+func (w *window) SetOnDropped(dropped func(pos fyne.Position, items []fyne.URI)) {
+	runOnDraw(w, func() {
```

Comment:
It seems like the passed function that I want to run on the draw thread never actually runs. The code within works fine if I put it within one of the other functions that already are passed to `runOnDraw()` in other places within the codebase. Any help on why this queue never gets us anywhere would be helpful.

Reply:
Probably you shouldn't manually get GLFW contexts and issue calls. Normally the GLFW calls get issued on the main not the draw thread - draw is OpenGL context, main is the GLFW context. `w.viewport.SetDropCallback` should be the right way to call it...